### ISSUE TITLE
UncaughtReferenceError when choosing device download settings.

### ISSUE THREAD

**Issue by adketuri** [brianrak/satori-reader-mobile-beta#9]
**Describe the bug**
After logging in on my Android device, I am taken to a screen to choose download options for articles. When selecting the download favorited option and clicking "Save Settings", an error popup appears: 

> "An error was detected. Message: Uncaught ReferenceError: error is not defined Line number: 197"

This popup did not appear when I chose to not save to device.

**To Reproduce**
1. Open the App
2. Select log in with a new user
3. Enter valid email/password and select "log in"
4. When prompted for device settings (don't save to device, save starred to device, or save all to device), select save starred to device.
5. Select okay
6. Observe the error message.

**Expected behavior**
User should be taken to home screen with the starred series favorited.

**Screenshots**
![screenshot_20180625-163432](https://user-images.githubusercontent.com/2700286/41881158-dc12631e-7896-11e8-828e-9c03c94aab77.jpg)

**Smartphone (please complete the following information):**
 - Device: Android (OnePlus 3T)
 - OS: 8.0.0

**Additional context**
Likely irrelevant, but I recently removed the social login and opted to log in with a traditional username/password. I do have one series favorited from the web (Kona).

**Comment by MelissaMarie12** [comment]
I received the same exact bug when choosing favorite series as well. The error number was also error 197.

**Comment by wtn** [comment]
On iOS, I chose the “download favorited series” option and I got a different error message:
```
An error was detected. Message: Script error. Line number: 0
```

**Comment by brianrak** [comment]
Fixed in Build 0.906. Thank you, everyone!
